Mixpanel/MixpanelPersistance.cs

```diff
@@ -173,18 +172,31 @@ internal static Value TimedEvents
                 PlayerPrefs.SetString(TimedEventsName, JsonUtility.ToJson(_timedEvents));
             }
         }
-        
+
         #endregion
 
         #region TrackQueue
 
-        public static readonly PersistentQueue TrackQueue = new PersistentQueue(Path.Combine(Application.persistentDataPath, "mixpanel_track_queue"));
+#if UNITY_EDITOR
```

Comment:
This was added just for local testing. Because queue file is locked in play mode it's impossible to run two or more instances of unity with same project (but different target platform). Our game has PvP matches and it's very important to test it before real build, so only option for now it run two Unity with different target platform.
I change this part to use  EditorUserBuildSettings.activeBuildTarget when it runs form Editor. in real game it will use unmodified name.

Reply:
Ahh ok that makes alot of sense.  I'm going to dig on this a big and see if there is a better way to do this.  If not this should suffice.